make beeping optional in Screen.ShowHelpTextThisFrame()

DavidLiuGit opened the pull request:
Added a parameter in `Screen.ShowHelpTextThisFrame()` to optionally suppress beeping sound when showing help text.

Motivation:  
I'd like to display elapsed time of a [race](https://github.com/DavidLiuGit/GTAV_Lap_Timer) in real time without the constant beeping sound.

Also, the help text lingers on screen for 3 seconds. According to [NativeDB](http://www.dev-c.com/nativedb/func/info/238ffe5c7b0498a6), the last param dictates how long the help text is shown. The current value `-1` is the default, which shows the help text for 3 seconds.

crosire commented:
Modifying the existing overload is not a good option, since that breaks existing scripts using that function (since the method signature changes, so method lookup fails).
You can work around this by just adding an entirely new overload (`public static void ShowHelpTextThisFrame(string helpText, bool beep) {}` without the default parameter).

DavidLiuGit commented:
Good point. I hadn't thought about method signatures in compiled scripts.

I added an overloaded `ShowHelpTextThisFrame` with the new `bool beep` parameter. Also updated the logic in the original method to call the new method with `beep=true`, to keep the code DRY
